cloudformation/stack_topic_handler.js

```diff
@@ -228,3 +362,135 @@ exports.handler = async function (event, context) {
   }
 };
 
+// Assigned host names
+const ADJECTIVES = [
+  "admiring",
+  "adoring",
+  "affectionate",
+  "agitated",
+  "amazing",
+  "angry",
+  "awesome",
+  "blissful",
+  "boring",
+  "brave",
+  "clever",
+  "cocky",
+  "compassionate",
+  "competent",
+  "condescending",
+  "confident",
+  "cranky",
+  "dazzling",
+  "determined",
+  "distracted",
+  "dreamy",
+  "eager",
+  "ecstatic",
+  "elastic",
+  "elated",
+  "elegant",
+  "eloquent",
+  "epic",
+  "fervent",
+  "festive",
+  "flamboyant",
+  "focused",
+  "friendly",
+  "frosty",
+  "gallant",
+  "gifted",
+  "goofy",
+  "gracious",
+  "happy",
+  "hardcore",
+  "heuristic",
+  "hopeful",
+  "hungry",
+  "infallible",
+  "inspiring",
+  "jolly",
+  "jovial",
+  "keen",
+  "kind",
+  "laughing",
+  "loving",
+  "lucid",
+  "mystifying",
+  "modest",
+  "musing",
+  "naughty",
+  "nervous",
+  "nifty",
+  "nostalgic",
+  "objective",
+  "optimistic",
+  "peaceful",
+  "pedantic",
+  "pensive",
+  "practical",
+  "priceless",
+  "quirky",
+  "quizzical",
+  "relaxed",
+  "reverent",
+  "romantic",
+  "sad",
+  "serene",
+  "sharp",
+  "silly",
+  "sleepy",
+  "stoic",
+  "stupefied",
+  "suspicious",
+  "tender",
+  "thirsty",
+  "trusting",
+  "unruffled",
+  "upbeat",
+  "vibrant",
+  "vigilant",
+  "vigorous",
+  "wizardly",
+  "wonderful",
+  "xenodochial",
+  "youthful",
+  "zealous",
+  "zen",
+];
+
+const NOUNS = [
+  "ardent",
+  "artificer",
+  "balrog",
+  "barbarian",
+  "bard",
+  "cleric",
+  "druid",
+  "dwarf",
+  "elf",
+  "ent",
+  "fighter",
+  "giant",
+  "goblin",
+  "halfling",
+  "hobbit",
+  "illusionist",
+  "invoker",
+  "mage",
+  "monk",
+  "mystic",
+  "orc",
+  "paladin",
+  "psion",
+  "ranger",
+  "rogue",
+  "seeker",
+  "sorcerer",
+  "thief",
+  "troll",
+  "vampire",
+  "warlock",
+  "werewolf",
+  "wizard",
+];
```

Comment:
Having to include the list of adjectives and nouns here, and using them to identify records feels pretty fragile. If there's a better option we should at least open a ticket to explore it in the future. 

You may also need to worry about what will happen when this is minified and included in the stack template. Dunno if these lists will take us over some limit for hardcoded strings in the template, or template size in general.

Reply:
This is the "hacky" part I'm not happy about; I agree it's fragile _and_ it doesn't reference the original list since the list is hosted in a different repo. 

I couldn't find a way to extract the terminated server name or a reliable way to identify an assigned host name vs one the user added (something I'm doing to experiment with in separate project, so this is top of mind for me). I'll make a ticket to explore in the future if this gets merged.